pkg/promunifi/uap.go

```diff
@@ -300,7 +302,7 @@ func (u *promUnifi) exportVAPtable(r report, labels []string, vt unifi.VapTable)
 func (u *promUnifi) exportRadtable(r report, labels []string, rt unifi.RadioTable, rts unifi.RadioTableStats) {
 	// radio table
 	for _, p := range rt {
-		labelR := append([]string{p.Name, p.Radio}, labels[6:]...)
+		labelR := append([]string{p.Name, p.Radio, labels[1], labels[2]})
```

Comment:
same. `append()` not needed here.

Reply:
the build actually failed because of this. neato!  Haven't seen that one yet haha

Reply:
Fixed